src/main/java/frc/robot/Robot.java

```diff
@@ -40,6 +41,8 @@ public void robotInit() {
     this.robotContainer = new RobotContainer();
 
     datalog.dataLogRobotContainerInit(this.robotContainer);
+
+    SmartDashboard.putData(CommandScheduler.getInstance());
```

Comment:
Repeating comment from previous pull request. If possible, move this to into dataLogRobotContainerInit. If it isn't that is okay, but I would prefer to have datalogging isolated to datalogging.java, and putData calls in RobotContainer.java or in the subsystems.

Reply:
Since this seems to have been resolved in the other PR, I will resolve this convo.